### DISCUSSION TITLE
HTTP requests are not scaling up and scaling down

### DISCUSSION THREAD

**Discussion by Manikiran88888** [kedacore/http-add-on#994] [Q&A]
I am using tis code and the pods are not scaling up and scaling down even though I have given a high load 

This code which I have used is as fallows 

kind: HTTPScaledObject
apiVersion: http.keda.sh/v1alpha1
metadata:
    name: nginx-http-scaledobject
spec:
    host: myhost.com
    targetPendingRequests: 1
    scaleTargetRef:
        deployment: nginx
        service: nginx
        port: 80
    replicas:
        min: 1
        max: 3
        
 The scaled objects are also being assigned properly 
 
![image](https://github.com/kedacore/http-add-on/assets/86592911/14c256de-baed-44a7-93df-c57dc8de9572)

Is there any way to scale up the pods based on the http requests

**Comment by linkvt** [comment]
Closing this discussion as I'm doing a cleanup right now, if this is a bug that still exists please create an issue, otherwise just let me know.
